packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/OVirtVirtualMachinesRow.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { TableCell } from 'src/modules/Providers/utils';
+
+import { EnumToTuple, ResourceFieldFactory, RowProps } from '@kubev2v/common';
+import { OVirtVM } from '@kubev2v/types';
+import { Tr } from '@patternfly/react-table';
+
+import { VMCellProps } from './components';
+import { defaultCellRenderers, renderTd, VmData } from './ProviderVirtualMachinesRow';
+
+export const oVirtVmFieldsMetadataFactory: ResourceFieldFactory = (t) => [
+  {
+    resourceFieldId: 'name',
+    jsonPath: '$.name',
+    label: t('Name'),
+    isVisible: true,
+    isIdentity: true, // Name is sufficient ID when Namespace is pre-selected
+    filter: {
+      type: 'freetext',
+      placeholderLabel: t('Filter by name'),
+    },
+    sortable: true,
+  },
+  {
+    resourceFieldId: 'cluster',
+    jsonPath: '$.vm.cluster',
+    label: t('Cluster'),
+    isVisible: true,
+    isIdentity: false,
+    filter: {
+      type: 'freetext',
+      placeholderLabel: t('Filter by cluster'),
+    },
+    sortable: true,
+  },
+  {
+    resourceFieldId: 'host',
+    jsonPath: '$.vm.host',
+    label: t('Host'),
+    isVisible: true,
+    isIdentity: false,
+    filter: {
+      type: 'freetext',
+      placeholderLabel: t('Filter by host'),
+    },
+    sortable: true,
+  },
+  {
```

Comment:
thinking of @ahadas comment , I think we should move the concerns to be by default the 2nd column (and user can change it later)

note, current concern renderer and filter by concern is very "simple/generic" you may need to re-think the filtering and rendering of concerns, or we can leave it to a followup